finance.py

```
@@ -202,26 +202,23 @@ def read_pickle(self, obj):
     def dcf_test(self, stock):
         if stock.is_undervalued():
             return stock
-            #self.___undervalued_stocks.append(stock)
+
+    def dcf_compute(stock_obj_list):
+        tmp1 = self.___pool.map(self.dcf_test, stock_obj_list)
```

Ruff (0.11.9)

Undefined name `self`

(F821)